### ISSUE TITLE
crash after click connect button

### ISSUE THREAD

**Issue by leeyo** [jgh-/VideoCore#206]
[_session startRtmpSessionWithURL:@"rtmp://live-push.talk-fun.com/live" andStreamKey:@"sample"];

I just modify the rtmp url and run,then I got a crash.

###LOG###

2015-10-09 09:59:34.553 SampleBroadcaster[7325:1018319] Creating context
2015-10-09 09:59:34.586 SampleBroadcaster[7325:1018319] Context creation succeeded
Apple::AudioMixerplayPath: sample, app: liveClientState: 1
ClientState: 2
ClientState: 3
ClientState: 4
ClientState: 5
ClientState: 6
received server window size: 2500000
received peer bandwidth limit: 2500000 type: 2
received ping, sending pong.
Received invoke
pktId: 1
received invoke _result
tracked command: connect
ClientState: 7
received unknown packet type: 0x15
Received invoke
pktId: 4
received invoke _result
tracked command: createStream
ClientState: 8
received ping, sending pong.
Received invoke
pktId: 0
received invoke onStatus

**Comment by dourgulf** [comment]
Currently the RTMP code do not support multiple chunk case. If you don't want to fixed the code, you can try to modify the server setting by enlarge the chunk size to 1024.

**Comment by leeyo** [comment]
@dourgulf     How to fix the code ? i have no idea now

**Comment by dourgulf** [comment]
Update the lastest code please, rewrited RTMP parser had solved this problem.

**Comment by omarojo** [comment]
@dourgulf Hi, is this fix reflected in the cocoapod version? how can we get the latest version that has this fix. There are several other people having similar problem, https://github.com/jgh-/VideoCore/issues/148

**Comment by dourgulf** [comment]
@omarojo Yes, the rewrited RTMP parser already merge into the master. just update. Just need to notice the issue #225

**Comment by satyamub** [comment]
@dourgulf I am using Videocore 0.3.2 still I am facing the same issue. As seeing this, I have also changed the OutChunkSize of my server to 1024 and still not resolved. Did you resolve the issue?

**Comment by satyamub** [comment]
How can I update, I even tried to change the outchunksize to 512 to check for the issue still Its not working!!

**Comment by bintu1234** [comment]
I am also facing same issue can any body solved please help me

**Comment by dourgulf** [comment]
Update to last version of master, don't use 0.3.2 branch.

**Comment by satyamub** [comment]
@dourgulf how do I update..I am using COCOAPODS!!Please let me know?

**Comment by satyamub** [comment]
@bintu1234 did you understand what @dourgulf  said to update to latest version of master not branch?

**Comment by satyamub** [comment]
@omarojo how did you resolve the issue? how did you update?

**Comment by dourgulf** [comment]
@satyamub 
download source code from github, and:

cd sample/SampleBroadcaster
pod install
open SampleBroadcaster.xcworkspace

**Comment by dourgulf** [comment]
Sorry, I dont maintain the COCOAPODS, so can't help.

**Comment by satyamub** [comment]
@dourgulf I resolved the issue with version of 0.1.12.. its working however some times it starts broadcasting and some times it not!!!Can you tell me why?

**Comment by depthlove** [comment]
I met the same problem "crash after click connect button"

**Comment by m8282823** [comment]
me too ,who can help me ,my god!

**Comment by satyamub** [comment]
@m8282823  what is the red5 version you are using and also let me know the cocoapod version of video core you are using?

**Comment by m8282823** [comment]
what is red 5? the cocoa pod version of video core is 0.3.2   you had solved it  lucky boy ?

**Comment by m8282823** [comment]
the red5 version is 1.06

**Comment by satyamub** [comment]
videocore cocoapod version?

**Comment by m8282823** [comment]
version of videocore in cocoapods     is 0.3.2      the version of cocoa pods is 0.38.2   sorry for my English

**Comment by satyamub** [comment]
Can you change the red5 version to 1.0.3 and change to 0.1.12 videocore in cocoa pods
 and then try?

**Comment by m8282823** [comment]
what a old version , my server workmate is not agree - -
